src/sbt-test/sbt-buildinfo/append/build.sbt

```diff
@@ -12,47 +12,46 @@ lazy val root = (project in file("."))
     name := "helloworld",
     scalacOptions ++= Seq("-Xlint", "-Xfatal-warnings", "-Yno-imports"),
     buildInfoKeys ++= Seq[BuildInfoKey](name, organization, version, scalaVersion,
-      libraryDependencies, libraryDependencies in Test),
+      libraryDependencies, (Test / libraryDependencies): SettingKey[Seq[ModuleID]]),
```

Comment:
Scala 3 fails at inferring the type of `Test / libraryDependencies` because of double implicit conversion.